src/main/webapp/app/text-assessment/text-assessment.component.html

```diff
@@ -50,6 +50,14 @@ <h5 class="text-white">
             >
                 Override Assessment
             </button>
+            <!--the implementation to display this button should be changed if the condition to display this button is not same as override assessment -->
```

Comment:
I don't understand the comment. Is this a todo? Could be that I'm not deep enough into this topic, so maybe someone else can give feedback.

Reply:
No, it is not a todo. I think the comment was too vague. I have changed the comment. I hope the new comment is a bit more understandable